content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -110,11 +119,15 @@ pod in this list.
 
 To run multiple-scheduler with leader election enabled, you must do the following:
 
-First, update the following fields in your YAML file:
+First, update the scheduler configuration (earlier you applied a
```

Comment:
See my [comment about scheduler configuration](https://github.com/kubernetes/website/pull/30107/files#r740745459).
In either case, we should keep the case the same on this page as 
>Scheduler Configuration

is upper camel case in other parts of the page

Reply:
@reylejano already updated with https://github.com/kubernetes/website/pull/30107/commits/c5cb03ab13e4e101875e03a61689cad458ec782f.